### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by ricardocosta89**
Include support for running both the tests and the app itself using Docker containers.

### CODE REVIEW

**Comment by ricardocosta89** [comment]
@trenpixster Can you take a look, please? 🙏

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM node:7-slim
```
Comment (trenpixster) [inline Dockerfile]: Is this bundling yarn already?

Comment (ricardocosta89) [inline Dockerfile]: It is. 👍

**Comment by trenpixster** [comment]
LGTM 👍 🙇

**Pull request merged**